### DISCUSSION TITLE
How Besu is compared to GoQuorum?

### DISCUSSION THREAD

**Discussion by baptiste-b-pegasys** [ConsenSys/quorum#1293] [Q&A] [answered]
How Besu is compared to GoQuorum?

**Comment by baptiste-b-pegasys** [comment, marked as answer]
At a high level, the similarities and differences between Besu and GoQuorum are:

- Besu and GoQuorum are both Ethereum clients and implementations of the EEA client spec, adding additional privacy, permissioning and consensus configurations to Ethereum. Besu maintains full compatibility with the Ethereum mainnet.
- Besu is written in Java and Apache 2.0 licensed and GoQuorum is a fork of Go-Ethereum and written in Go with a LGPL license.
- The APIs and configuration options are largely similar but not a drop-in replacement for each other.
- The clients do not share any code except that they use the same private transaction manager, Tessera. We will be investing more in Besu over the longer term which is why we encourage it for new networks.
- Besu supports gas fee, GoQuorum does not.

**Comment by rafhofman** [comment]
Hello, @baptiste-b-pegasys - how it looks in July 2023? Considering private networks, with few req/s operations and heavy reads. Thanks!
